packages/@unimodules/core/ios/UMCore/Protocols/UMJavaScriptContextProvider.h

```diff
@@ -6,5 +6,6 @@
 @protocol UMJavaScriptContextProvider <NSObject>
 
 - (JSGlobalContextRef)javaScriptContextRef;
+- (long)javaScriptRuntimePtr;
```

Comment:
Seems like we should use `void *` since `long` and `jlong` can be different and when we pass javaScriptRuntimePtr into other methods it gets treated as a pointer anyway.